libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-row-delete.directive.ts

```diff
@@ -199,6 +217,22 @@ export class SkyAgGridRowDeleteDirective
     }
   }
 
+  public ngAfterViewInit(): void {
+    this.#scrollableHostService
+      .watchScrollableHostClipPathChanges(this.#elementRef)
+      .pipe(takeUntil(this.#ngUnsubscribe))
+      .subscribe((clipPath) => {
+        this.#clipPath$.next(clipPath);
+      });
+
+    const zIndex = this.#stackingContextService.getZIndex(
+      this.#elementRef.nativeElement
+    );
+    if (zIndex) {
+      this.#zIndex.next(zIndex.toString(10));
+    }
```

Comment:
Using the new stacking context service to identify if the modal is in a stacked element e.g. modal so the overlay is on the same z-index.